cron/cron.ts

```diff
@@ -17,7 +17,7 @@ import { handleSendMessageError } from '#helper/errorHandler'
 async function monthly() {
   const now = new Date()
   const currentMonth = now.getMonth() + 1
-  const keyboardMessage = HLanguage('en', 'region')
+  const keyboardMessage = HLanguage('region')
   const regions = Object.keys(keyboardMessage)
   const regionIds = Object.values(keyboardMessage)
   const daysOfWeek = ['Якшанба', 'Душанба', 'Сешанба', 'Чоршанба', 'Пайшанба', 'Жума', 'Шанба']
```

Comment:
@Xayrulloh I think @eugene0928 talks about this variable
Why is it in krill alphabet?

Reply:
Pdf ga oi

Reply:
Oid

Reply:
are weekdays related to pdf?

Reply:
yes

Reply:
Did u see entire code of that file.
I didn't that's why I've to look for it to answer your question.

Reply:
I easily forget the codes I wrote

Reply:
nope just u send on discord

Reply:
I recommend